rpcd/etc/openstack_deploy/user_rpcm_variables.yml

```diff
@@ -0,0 +1,104 @@
+---
+# Copyright 2016-2017, Rackspace US, Inc.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+# MAAS
+# Set maas_auth_method to 'token' to use maas_auth_token/maas_api_url
+# instead of maas_username/maas_api_key
+maas_auth_method: password
+maas_tenant_id: "{{ rackspace_cloud_tenant_id }}"
+maas_auth_url: "{{ rackspace_cloud_auth_url }}"
+maas_username: "{{ rackspace_cloud_username }}"
+maas_api_key: "{{ rackspace_cloud_api_key }}"
+maas_auth_token: some_token
```

Comment:
why here is maas_auth_token: some_token instead of "{{ rackspace_cloud_auth_token | default('') }}"